src/openrct2/ride/Vehicle.cpp

```diff
@@ -6067,9 +6067,10 @@ void Vehicle::ApplyNonStopBlockBrake()
             velocity = BLOCK_BRAKE_BASE_SPEED;
             acceleration = 0;
         }
-        else
+        // the 868 offset is to make the min speed equal to
+        // BLOCK_BRAKE_BASE_SPEED to keep existing behaviour
```

Comment:
Why not use the `BLOCK_BRAKE_BASE_SPEED` constant then?

Reply:
It would be messy if I used `BLOCK_BRAKE_BASE_SPEED` in the equality:
```cpp
else if (velocity > (brake_speed << 16) + (BLOCK_BRAKE_BASE_SPEED - (RCT2DefaultBlockBrakeSpeed << 16))
```

I made a constant so it's no longer a magic number.